opendatahub-io/odh-manifests
# remove external istio crds from odh-model-controller manifests

Fix odh-model-controller expecting istio `PeerAuthentication` and `Telemetry` CRDs even when they are not needed for modelmesh. 
Intended to be tested with the accompanying PR to [odh-model-controller](https://github.com/opendatahub-io/odh-model-controller/pull/68)

Note: the e2e tests on this PR will fail till https://github.com/opendatahub-io/odh-model-controller/pull/68 is merged into the release branch 

## How Has This Been Tested?
- Install ODH operator from the `stable` channel. 
- Copy the [odh-core kfdef](https://github.com/opendatahub-io/odh-manifests/blob/master/kfdef/odh-core.yaml) 
- Change the manifest uri in the kfdef to `https://github.com/VedantMahabaleshwarkar/odh-manifests/tarball/crd_fix` 
- Install the kfdef 
- Create a OVMS model and verify it is successfully created
- Verify odh-model-controller is not Crashlooping 

## Merge criteria:
